src/mito2/src/region/version.rs

```diff
@@ -287,6 +340,8 @@ impl VersionBuilder {
             ssts: self.ssts,
             flushed_entry_id: self.flushed_entry_id,
             flushed_sequence: self.flushed_sequence,
+            truncate_entry_id: self.truncate_entry_id,
```

Comment:
We also need to recover them in opener like flushed entry id.
https://github.com/GreptimeTeam/greptimedb/blob/50cea29b29a45073a11b1cb3702a5eaa21b465cb/src/mito2/src/region/opener.rs#L154-L155